00-Basics/03-AllDifferent/alldifferentcpp.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+
+
+int main()
+{
+    int x, y, z;
+    
+    cout << "Enter the three numbers : ";
+    cin >> x >> y>>z;
+    if(x==y ||x==z ||y==z ){
+    	cout << "not all different ";
+	}
+	else{
+		cout << "all different";
+	}
+	return 0;
+}
```

Comment:
Stick with a style for all of the source codes. You either always open `{` in the same line for if, or always open it in the next line. Both of them are good, but stick to one.

Also, indentation. I don't know why this is always a problem. And I don't know what kind of code editor you use, but modern ones can already do this for you. For such small exercises, there are online tools as well:

https://codebeautify.org/cpp-formatter-beautifier

this will give for example this output:
```
int main() {
  int x, y, z;

  cout << "Enter the three numbers : ";
  cin >> x >> y >> z;
  if (x == y || x == z || y == z) {
    cout << "not all different ";
  } else {
    cout << "all different";
  }
  return 0;
}
```

But really, vscode, Qtcreator, visual studio, eclipse, etc. I'm sure have plugins to help with this.